LoanGurus/partner-portal
# DEV-1755: Send referral emails

This adds a call to send referral emails during the referrals onboarding
step.

[changelog]
added: DEV-1755: Send referral emails

TO TEST:
On the referral onboarding step, add an email address you have access to.  After submit, check your spam filter for an email from Matt Willison.  Click the button for "Enroll in partner program".  You should be taken to a page with your invitation ID in the URL (for example: https://partner.loangurusdev.com/sign-up?invitation_id=85c0239b-86f1-483e-9e7c-01818454218a).  Compare it to the invite in the partner db => partner_invites.id

ps-id: CE30DC3E-868F-4AB1-87B0-0B651E66D888
